.travis.yml

```diff
@@ -7,19 +7,20 @@ jobs:
   fast_finish: true
   include:
     # macOS build
-    - env: INSTALL_TYPE=develop FITS="astropy" TEST=submodule MATPLOTLIBVER="3" XSPECVER="12.10.1n" TRAVIS_PYTHON_VERSION="3.7"
+    - env: INSTALL_TYPE=develop FITS="astropy" TEST=submodule MATPLOTLIBVER="3" XSPECVER="12.10.1n" TRAVIS_PYTHON_VERSION="3.8"
       os: osx
     # Barebone build to check tests pass when most of the tests must be skipped.
     # We need to use TEST=none to remove the test-data submodule.
     # This used to be the Python 2.7 build (so used an old NumPy) but it has now
     # been bumped to "nearly latest" versions
     - env: INSTALL_TYPE=develop TEST=none NUMPYVER="1.17" TRAVIS_PYTHON_VERSION="3.8"
-    # Full build (including ds9 and xspec), Python 3.5, setup.py develop, astropy, matplotlib 2
-    - env: XSPECVER="12.10.1n" NUMPYVER="1.11" FITS="astropy pytest-openfiles<=0.4.0 pytest-doctestplus<=0.5.0" INSTALL_TYPE=develop TEST=submodule MATPLOTLIBVER="2" TRAVIS_PYTHON_VERSION="3.5"
+    # Full build (including ds9 and xspec), Python 3.6, setup.py develop, astropy, matplotlib 2
     # As above, Python 3.6, setup.py install, xspec 12.10
     - env: XSPECVER="12.10.1n" NUMPYVER="1.11" FITS="astropy" INSTALL_TYPE=install TEST=submodule MATPLOTLIBVER="2" TRAVIS_PYTHON_VERSION="3.6"
     # As above, python 3.7, numpy 1.15, matplotlib 3
     - env: XSPECVER="12.10.1n" NUMPYVER="1.15" FITS="astropy" INSTALL_TYPE=install TEST=submodule MATPLOTLIBVER="3" TRAVIS_PYTHON_VERSION="3.7"
+    # As above, python 3.8, numpy 1.19, matplotlib 3
+    - env: XSPECVER="12.10.1n" NUMPYVER="1.19" FITS="astropy" INSTALL_TYPE=install TEST=submodule MATPLOTLIBVER="3" TRAVIS_PYTHON_VERSION="3.7"
```

Comment:
@DougBurke Why do you have `TRAVIS_PYTHON_VERSION="3.7"` here? Should this be 3.8?

Reply:
Doh. I'm less bothered about the linux 3.8 build though than the macOS one. This is just to have some evidence for a fix for whenever anyone gets around to fixing #730